src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1627,6 +1627,50 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
           $ref: '#/components/responses/defaultResponse'
+  /groups/{id}/user:
```

Comment:
If the endpoint is adding a single user to the group at a time, why not add it to the existing endpoint URL `DELETE /groups/{id}/user/{uid}` as a `POST` request?

Reply:
sure done

Reply:
Thanks

Reply:
welcome